src/swap/types.ts

```diff
@@ -59,6 +69,26 @@ export type QuoteWarning = {
   type?: string; // The type of the warning
 };
 
+export type RawTransactionData = {
+  data: string; // The transaction data
+  from: string; // The sender address
+  gas: string; // The gas limit
+  gasPrice: string; // The gas price
+  to: string; // The recipient address
+  value: string; // The value of the transaction
+};
```

Comment:
I feel like we should be able to grab all types like this from viem? To build our intuition, if we're _ever_ building custom utilities for core functionality (eg transaction types), then I think we should double check nothing from viem/wagmi can serve us.